ra2ce/graph/network_config_data/network_config_data_reader.py

```diff
@@ -153,8 +160,14 @@ def get_network_section(self) -> NetworkSection:
         _network_section.save_gpkg = self._parser.getboolean(
             _section, "save_gpkg", fallback=_network_section.save_gpkg
         )
-        _network_section.road_types = self._parser.getlist(
-            _section, "road_types", fallback=_network_section.road_types
+        _network_section.network_type = NetworkTypeEnum.get_enum(
+            self._parser.get(_section, "network_type", fallback=None)
+        )
+        _network_section.road_types = list(
+            map(
+                RoadTypeEnum.get_enum,
+                self._parser.getlist(_section, "road_types", fallback=[None]),
```

Comment:
The default value should be `[]`?
```suggestion
                self._parser.getlist(_section, "road_types", fallback=[]),
```

Reply:
Adjusted